react/react1/week2/react1week2homework/src/components/TodoList.js

```diff
@@ -25,7 +25,7 @@ function TodosMain() {
   }
 
   //todoList
-  const todoList = todos.map((todo) => (
+  const todoItem = todos.map((todo) => (
     <TodoItem
```

Comment:
You renamed it todoItem from todoList, but what this function returns is the todo list mapped. So it returns an array of objects, where each object is a <TodoItem/>. So maybe it is the todoList variable afterall?